src/Relude/Extra/Foldable.hs

```diff
@@ -38,3 +40,18 @@ foldlSC f = flip $ foldr go id
         Left l  -> l
         Right r -> k r
 {-# INLINE foldlSC #-}
+
+{- | Compute average of 'Foldable'
+>>> average1 ([1,2,3,4])
+2.5
```

Comment:
Shouldn't it be just `average` in this module?
```suggestion
>>> average [1,2,3,4]
Just 2.5
```

Reply:
Yeah. That's what I get by multitasking